support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.ts

```diff
@@ -135,7 +131,7 @@ export type RegularPaymentFields =
 	| RegularAmazonPayPaymentFields;
 export type RegularPaymentRequestAddress = {
 	country: IsoCountry;
-	state?: UsState | CaState | null;
+	state?: UsState | null;
```

Comment:
I wonder if there'd be a way to collapse the `null` and `undefined` here? Do they have different meanings? (Probably something for a future PR!)

Reply:
Yea, there definitely is. It would be useful to have an initiate to have 1 `None` value (`undefined` please).

I've left this outside of the scope of this PR as it's quite far reaching as is and there might be some check somewhere that breaks in different conditions.